src/AnnenErfaring/Seksjon.elm

```diff
@@ -107,34 +107,42 @@ type alias BeskrivelseInfo =
     }
 
 
-type alias FraDatoInfo =
+type alias FraÅrInfo =
     { rolle : String
     , beskrivelse : String
-    , fraMåned : Måned
     , fraÅr : String
     , tillatÅViseFeilmeldingÅr : Bool
     }
 
 
-type alias ValidertFraDatoInfo =
+type alias FraDatoInfo =
     { rolle : String
     , beskrivelse : String
     , fraMåned : Måned
     , fraÅr : År
     }
 
 
-type alias TilDatoInfo =
+type alias TilÅrInfo =
     { rolle : String
     , beskrivelse : String
     , fraMåned : Måned
     , fraÅr : År
-    , tilMåned : Måned
     , tilÅr : String
     , tillatÅViseFeilmeldingÅr : Bool
     }
 
 
+type alias TilDatoInfo =
+    { rolle : String
+    , beskrivelse : String
+    , fraMåned : Måned
+    , fraÅr : År
+    , tilMåned : Måned
```

Comment:
Jeg tror kanskje ikke du egentlig trenger `tilMåned` her? For du initierer den jo til Januar før brukeren har trykket på noen knapp, og når brukeren trykker på en knapp går man jo direkte videre til å ha et validert skjema. Samme med FraDatoInfo, tror ikke dn helle trenger å ha måned.